pennylane/ops/qutrit/matrix_ops.py

```diff
@@ -316,7 +317,8 @@ def pow(self, z):
         return super().pow(z)
 
     def _controlled(self, wire):
-        ctrl_wires = sorted(self.control_wires + wire)
-        ControlledQutritUnitary(
+        ctrl_wires = self.control_wires + wire
+        new_op = ControlledQutritUnitary(
```

Comment:
Just taking a look at the [qubit case](https://github.com/PennyLaneAI/pennylane/blob/d565eec3c22d45aff1bec1029134aafe0d4f50bb/pennylane/ops/qubit/matrix_ops.py#L378), I think we will need to update the `control_values` as well here to ensure that they are preserved when the new operation is constructed. This change will also need a corresponding test.